sql/catalyst/src/main/java/org/apache/spark/sql/connector/ExternalCommandRunner.java

```diff
@@ -27,6 +27,8 @@
  * cores for Solr and so on.
  * <p>
  * This interface will be instantiated when end users call `SparkSession#executeCommand`.
+ *
+ * @since 3.0.0
```

Comment:
Oh..no...thanks for adding this. BTW, how do you find it?

Reply:
I compare the latest Scala API doc against the 2.4 version. :)